pkgs/applications/misc/remarkable/reStream/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, bash
+, stdenv
+, lz4
+, ffmpeg-full
+, fetchFromGitHub
+, openssh
+}:
+
+stdenv.mkDerivation rec {
+  pname = "reStream";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "rien";
+    repo = pname;
+    rev = version;
+    sha256 = "18z17chl7r5dg12xmr3f9gbgv97nslm8nijigd03iysaj6dhymp3";
+  };
+
+  buildInputs = [ ffmpeg-full lz4 bash openssh ];
+
+  dontConfigure = true;
+  dontBuild = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    # make the output directory
+    mkdir -p $out/bin
+
+    # copy the reMarkable ARM binary to the output directory
+    cp ${src}/restream.arm.static $out/bin
```

Comment:
That doesn't make sense: `restream.arm.static` shouldn't end up in `$PATH`.

Reply:
Cool. Is there a correct way to ship the binary? Or should I just leave that out?

Reply:
What do other distros do? Maybe in `$out/share` somewhere?

Reply:
I do not know.